### PR TITLE
Upgrade to 2020-05-05

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Security alert*

## Solution
- *switch to last upstream version*
- *apply last example_ynh*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/hextris_ynh%20PR19/)  

When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -28,10 +29,10 @@ app=$YNH_APP_INSTANCE_NAME
 #=================================================
 # CHECK IF THE APP CAN BE INSTALLED WITH THESE ARGS
 #=================================================
-ynh_script_progression --message="Validating installation parameters..."
+ynh_script_progression --message="Validating installation parameters..." --weight=1
```
Comment (Aeris1One) [inline scripts/install:32]: --weight=1 is useless, isn't it ?

File: scripts/install [hunk 2]
```diff
@@ -48,19 +49,18 @@ ynh_app_setting_set --app=$app --key=is_public --value=$is_public
 #=================================================
 # STANDARD MODIFICATIONS
 #=================================================
-# COPY SOURCES
+# DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
-ynh_script_progression --message="Setting up source files..."
+ynh_script_progression --message="Setting up source files..." --weight=1
```
Comment (Aeris1One) [inline scripts/install:54]: Same

File: scripts/install [hunk 3]
```diff
@@ -70,14 +70,15 @@ ynh_add_nginx_config
 #=================================================
 # SECURE FILES AND DIRECTORIES
 #=================================================
+ynh_script_progression --message="Securing files and directories..." --weight=1
```
Comment (Aeris1One) [inline scripts/install:73]: Same

File: scripts/install [hunk 4]
```diff
@@ -70,14 +70,15 @@ ynh_add_nginx_config
...
 
 # Set permissions to app files
 chown -R root: $final_path
 
 #=================================================
 # SETUP SSOWAT
 #=================================================
-ynh_script_progression --message="Configuring SSOwat..."
+ynh_script_progression --message="Configuring SSOwat..." --weight=1
```
Comment (Aeris1One) [inline scripts/install:81]: Same

File: scripts/remove [hunk 5]
```diff
@@ -6,12 +6,13 @@
...
 #=================================================
 
+source _common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
 # LOAD SETTINGS
 #=================================================
-ynh_script_progression --message="Loading installation settings..."
+ynh_script_progression --message="Loading installation settings..." --weight=1
```
Comment (Aeris1One) [inline scripts/remove:15]: Same

File: scripts/remove [hunk 6]
```diff
@@ -23,7 +24,7 @@ final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 #=================================================
 # REMOVE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Removing app main directory..."
+ynh_script_progression --message="Removing app main directory..." --weight=1
```
Comment (Aeris1One) [inline scripts/remove:27]: Same

File: scripts/restore [hunk 7]
```diff
@@ -29,7 +31,7 @@ final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 #=================================================
 # CHECK IF THE APP CAN BE RESTORED
 #=================================================
-ynh_script_progression --message="Validating restoration parameters..."
+ynh_script_progression --message="Validating restoration parameters..." --weight=1
```
Comment (Aeris1One) [inline scripts/restore:34]: Same

File: scripts/restore [hunk 8]
```diff
@@ -41,13 +43,14 @@ test ! -d $final_path \
 #=================================================
 # RESTORE THE NGINX CONFIGURATION
 #=================================================
+ynh_script_progression --message="Restoring the nginx configuration..." --weight=1
```
Comment (Aeris1One) [inline scripts/restore:46]: Same

File: scripts/restore [hunk 9]
```diff
@@ -41,13 +43,14 @@ test ! -d $final_path \
...
 
 ynh_restore_file --origin_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 
 #=================================================
 # RESTORE THE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Restoring the app main directory..."
+ynh_script_progression --message="Restoring the app main directory..." --weight=1
```
Comment (Aeris1One) [inline scripts/restore:53]: Same

File: scripts/restore [hunk 10]
```diff
@@ -56,7 +59,7 @@ ynh_restore_file --origin_path="$final_path"
 #=================================================
 # RELOAD NGINX
 #=================================================
-ynh_script_progression --message="Reloading nginx web server..."
+ynh_script_progression --message="Reloading nginx web server..." --weight=1
```
Comment (Aeris1One) [inline scripts/restore:62]: Same

File: scripts/upgrade [hunk 11]
```diff
@@ -78,15 +78,15 @@ path_url=$(ynh_normalize_url_path --path_url=$path_url)
...
 # STANDARD UPGRADE STEPS
 #=================================================
-# COPY SOURCES
+# DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
-	ynh_script_progression --message="Upgrading source files..."
+	ynh_script_progression --message="Upgrading source files..." --weight=1
```
Comment (Aeris1One) [inline scripts/upgrade:86]: Same

File: scripts/install (hunk 1 above)
Comment (yalh76) [reply scripts/install:32]: Yes it's completely useless as: 
https://github.com/YunoHost/yunohost/blob/8ecd8fe6a334e578709120765468a5e8cb6ebd23/data/helpers.d/logging#L235

But added to be consistent with other `ynh_script_progression` lines

**Pull request merged**